triton/kubernetes/AWS/FIL/README.md

```diff
@@ -0,0 +1,215 @@
+## Deploying a Triton service that uses a custom plugin for the RAPIDS cuML Forest Inference Library (FIL).
+
+**NOTE:** For steps to setup a horizontally autoscalable Triton Service in EKS refer to the instructions in [Detailed_HPA_Setup](./Detailed_HPA_Setup.md).
+
+### Overview
+
+This example will illustrate the workflow to deploy a Triton inference service, with a [non-trival custom backend](https://github.com/triton-inference-server/fil_backend) for accelerated forest model inference on [Amazon Elastic Kubernetes Service (EKS)](https://aws.amazon.com/eks/?whats-new-cards.sort-by=item.additionalFields.postDateTime&whats-new-cards.sort-order=desc&eks-blogs.sort-by=item.additionalFields.createdDate&eks-blogs.sort-order=desc). 
+
+It is assumed that you have an existing account with sufficient compute and GPU quota, a correctly configured `kubectl` utility, and a properly configured and accessible AWS S3 bucket.
+
+**NOTE:** When referring to configuration parameters or elements that are `specific to your environment`, they will be represented as linux-style environment variables ex. ${YOUR_CONFIG_PARAM}.
+
+ 
+Specific parameters used here include:
+
+- REGION_NAME= < your preferred location >
+- EKS_CLUSTER_NAME= < your cluster name >
+- INSTANCE_TYPE= g4dn.12xlarge # or any other VM size with GPUs.
+- AWS_ACCOUNT_ID= < select your ACCOUNT ID >
+- REPOSITORY_NAME= titon_fil_backend # < name of the custom image we are going to build for triton backend, you can change this to your preference >
+- S3_BUCKET_PATH= < s3 bucket name (existing or to be created) to host the models >
+
+  
+---
+
+### Step 0: Pre-requisites
```

Comment:
You can [drop the hyphen](https://www2.calstate.edu/csu-system/csu-branding-standards/editorial-style-guide/Pages/Commonly-Used-Words.aspx#P) here.